src/applications/ivc-champva/10-10D/pages/ApplicantRelationshipPage.jsx

```diff
@@ -8,9 +8,20 @@ import { titleUI } from 'platform/forms-system/src/js/web-component-patterns';
 import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
```

Comment:
Refactored a good bit of this to make it more flexible. Main change is that instead of having hardcoded object properties for `relationshipToVeteran` and `otherRelationshipToVeteran`, we can override them now and use whatever we want.

Reply:
In the near future, planning on renaming this to reflect it's more generic usage.